src/bacnet/basic/object/av.c

```diff
@@ -220,8 +202,6 @@ bool Analog_Value_Valid_Instance(uint32_t object_instance)
  */
 unsigned Analog_Value_Count(void)
 {
-    PRINT(" ");
-    //NMC4 TODO: we could modify this to only return number of values that we actually set.
```

Comment:
I would keep this comment

Reply:
This is invoked by the device as well when it computes the total number of objects of all types....

Reply:
yes, and we prob only want to "use" at run-time only the number which are required by that implementation, e.g. 14 AVs for App Alpha, not 32.

Reply:
Restored